src/main/java/com/example/delivery_system/controller/CarriageController.java

```diff
@@ -0,0 +1,43 @@
+package com.example.delivery_system.controller;
+
+import com.example.delivery_system.entity.Carriage;
+import com.example.delivery_system.service.CarriageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class CarriageController {
+    private CarriageService carriageService;
+
+    @Autowired
+    public CarriageController(CarriageService carriageService) {
+        this.carriageService = carriageService;
+    }
+
+    @GetMapping("/сarriages")
+    public List<Carriage> allCarriages() {
```

Comment:
Please, wrap a response of your endpoints in ResponseEntity to return the right response status code.

E.g.:
```
    @GetMapping("/сarriages")
    public ResponseEntity<List<Carriage>> allCarriages() {
        return ResponseEntity.ok(carriageService.allCarriages());
    }
```

```
    @DeleteMapping("/сarriages/{id}")
    ResponseEntity<Void> deleteCarriage(@PathVariable Long id) {
        carriageService.deleteById(id);
        return ResponseEntity.noContent().build();
    }
```

```
    @PostMapping("/сarriage")
    ResponseEntity <Carriage> newCarriage(@RequestBody Carriage newCarriage) {
        return ResponseEntity.status(HttpStatus.CREATED).body(carriageService.save(newCarriage));
    }
```